source/postprocess/visualization.cc

```diff
@@ -729,6 +744,11 @@ namespace aspect
                              "'Use years in output instead of seconds' parameter is set; "
                              "seconds otherwise.");
 
+          prm.declare_entry ("Steps between graphical output", boost::lexical_cast<std::string>(std::numeric_limits<int>::max()),
```

Comment:
I didn't read this carefully -- please call it "Time steps between graphical output" to make clear what kind of steps it is.